viewer/src/models/cad/determineSectors.ts

```diff
@@ -60,14 +60,7 @@ export async function defaultDetermineSectors(params: DetermineSectorsInput): Pr
 
     const screenHeight = 2.0 * distanceToCamera(sector) * Math.tan((cameraFov / 2) * degToRadFactor);
     const largestAllowedQuadSize = hints.maxQuadSize * screenHeight; // no larger than x percent of the height
-    const quadSize = (() => {
-      if (!sector.simple) {
-        // Making the quad infinite in size means we will always use the detailed version instead
-        return Infinity;
-      }
-      return sector.simple.gridIncrement;
-    })();
-
+    const quadSize = sector.facesFile.quadSize;
```

Comment:
Is there a quadSize even if the file does not exist?

Reply:
Yes, there is now. Quadsize is deterministic based on depth in the tree, and this is now output from the optimizer. We changed this so it's always provided - filename is set to null when the file isn't present. If quadsize wasn't provided for sectors without F3D, we would always have to resort to high detail geometry as there would be no way of determining if we can use F3D.